docs/user_guide/configuration.rst

```diff
@@ -102,11 +113,34 @@ Other
 On systems that have neither an ``%LOCALAPPDATA%`` or ``$HOME`` environment
 variable, global config files are not currently supported.
 
+Local Configuration Files
+-------------------------
+
+Local configuration files are also named ``idaes.conf`` and can be placed in the
+working directory, which is the directory you launch Python from.  You can also
+use the Python command ``chdir()`` to change the working directory before
+importing ``idaes``.
+
+In addition to reading local configuration files when ``idaes`` is imported, you
+can read a configuration file anytime by calling ``idaes.read_config(path)``.
+Reading a configuration file will automatically apply and resulting configuration
+changes.
+
+Changing the Configuration in a Script or Interactive Session
+-------------------------------------------------------------
+
+The idaes configuration can be changed anytime after the ``idaes`` module is
+imported.  The standard ConfigBlock options are described in detail below.  For
+example to change whether you want to use the solvers provided by idaes or ones
+you have installed elsewhere, you would first use the command
+``idaes.cfg["use_idaes_solvers"] = False`` then to make the change take effect
+use ``idaes.reconfig()``.  Not all option changes require ``idaes.reconfig()``,
+so whether they do or don't is provided in the options descriptions below.
 
 Important Configuration Entries
```

Comment:
I think you should list the default configuration setting for all of these entries